examples/resources/project_member_group.tf

```diff
@@ -0,0 +1,7 @@
+resource "doppler_project_member_group" "backend_engineering" {
+  project      = "backend"
+  group_slug   = doppler_group.engineering.slug
+  role         = "collaborator"
+  environments = ["dev", "stg"]
```

Comment:
nit: in this example, should the `project` and `environments` also reference a terraform resource? (like the `group_slug`)

Reply:
That SGTM!